public/app/features/explore/PrometheusListView/RawListContainer.tsx

```diff
@@ -0,0 +1,162 @@
+import { css } from '@emotion/css';
+import { cloneDeep } from 'lodash';
+import React, { useEffect, useRef, useState } from 'react';
+import { useWindowSize } from 'react-use';
+import { VariableSizeList as List } from 'react-window';
+
+import { DataFrame, Field } from '@grafana/data/src';
+import { Button, stylesFactory } from '@grafana/ui/src';
+
+import {
+  getRawPrometheusListItemsFromDataFrame,
+  RawPrometheusListItemEmptyValue,
+} from '../utils/getRawPrometheusListItemsFromDataFrame';
+
+import { ItemLabels } from './ItemLabels';
+import RawListItem from './RawListItem';
+
+export type instantQueryRawVirtualizedListData = { Value: string; __name__: string; [index: string]: string };
+
+export interface RawListContainerProps {
+  tableResult: DataFrame;
+}
+
+const getRawListContainerStyles = stylesFactory(() => {
+  return {
+    wrapper: css`
+      height: 100%;
+      overflow: scroll;
+    `,
+    mobileWrapper: css`
+      height: 100%;
+      overflow: scroll;
+    `,
+    header: css`
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      padding: 10px 0;
+    `,
+  };
+});
+
+const mobileWidthThreshold = 480;
+const numberOfColumnsBeforeExpandedViewIsDefault = 2;
+
+/**
+ * The container that provides the virtualized list to the child components
+ * @param props
+ * @constructor
+ */
+const RawListContainer = (props: RawListContainerProps) => {
+  const { tableResult } = props;
+  let dataFrame = cloneDeep(tableResult);
+  const styles = getRawListContainerStyles();
+  let listRef = useRef<List | null>(null);
```

Comment:
```suggestion
  const listRef = useRef<List | null>(null);
```

`listRef` is never reassigned